build.gradle

```diff
@@ -158,21 +135,25 @@ dependencyCheck {
 
 ext {
   log4JVersion = "2.17.0"
+  feignJackson = '11.6'
+  groovyVersion = '3.0.11!!'
+  hamcrestVersion = '1.3'
+  hibernateValidatorVersion = '6.0.20.Final'
+  jacksonVersion = '2.13.2'
+  javaxValidationVersion = '2.0.1.Final'
```

Comment:
hibernateValidatorVersion, jacksonVersion and javaxValidationVersion don't appear to be used anywhere - should they still be included?

Reply:
these are needed as this will override the transitive dependencies versions from gradle parent modules

Reply:
I'm probably wrong, but I thought that these variables were just used within the build.gradle file and that the override variables were the '<dependency>.version' ones e.g. hibernate-validator.version for overriding the Hibernate validator version (see https://github.com/hmcts/ccd-definition-store-api/blob/master/build.gradle#L53).  Is this not the case?

Reply:
removed these now